trieste/acquisition/function.py

```diff
@@ -493,7 +497,7 @@ def prepare_acquisition_function(
             )
 
         constraint_fn = self._constraint_builder.prepare_acquisition_function(datasets, models)
-        pof = constraint_fn(objective_dataset.query_points)
+        pof = constraint_fn(objective_dataset.query_points[:, None, ...])
```

Comment:
expand dims looks cleaner?

Reply:
haha. artem actually moved me to index notation, which I think is clearer, cos you can see the final shape. Not too bothered though